app/Jobs/DatabaseBackupJob.php

```
@@ -74,13 +74,21 @@ public function __construct(public ScheduledDatabaseBackup $backup)
     }
 
     public function handle(): void
     {
         try {
+            $attempts = 0;
+            do {
+                $this->backup_log_uuid = (string) new Cuid2;
+                $exists = ScheduledDatabaseBackupExecution::where('uuid', $this->backup_log_uuid)->exists();
+                $attempts++;
+                if ($attempts >= 3 && $exists) {
+                    throw new \Exception('Unable to generate unique UUID for backup execution after 3 attempts');
```

PHPMD (2.15.0)

Missing class import via use statement (line '88', column '31'). (undefined)

(MissingImport)